transformations/gender_neopronouns/test.json

```diff
@@ -0,0 +1,41 @@
+{
+    "type": "gender_neopronouns",
+    "test_cases": [
+      {
+        "class": "GenderNeoPronouns",
+        "args" :{
+          "swap_neopronoun": "They"
+       },
+        "inputs": {
+          "sentence": "He went to the mall."
+        },
+        "outputs": [{
+          "sentence": "They went to the mall."
+        }]
+      },
+      {
+        "class": "GenderNeoPronouns",
+        "args" :{
+          "swap_neopronoun": "Xe"
+       },
+        "inputs": {
+          "sentence": "I met them today."
+        },
+        "outputs": [{
+          "sentence": "Xe met xem today."
+        }]
+      },
+      {
+        "class": "GenderNeoPronouns",
+        "args" :{
+          "swap_neopronoun": "Ey"
+       },
+        "inputs": {
+          "sentence": "I have to drive myself to school."
+        },
+        "outputs": [{
+          "sentence": "Ey have to drive emself to school."
```

Comment:
Can you add a test case with the third person singular s, like a transformation of "She walks to the mall" to "they"?